templates/web/base/report/new/category.html

```diff
@@ -15,7 +15,8 @@
     </label>[% =%]
     <select class="form-control[% IF category_groups.size %] js-grouped-select[% END %]" name="category" id="form_category"
     [%~ IF c.user.from_body =%]
-      data-role="[% c.user.has_body_permission_to('planned_reports') ? 'inspector' : 'user'  %]" data-body="[% c.user.from_body.name %]" data-prefill="[% c.cobrand.prefill_report_fields_for_inspector %]"
+      [%~ prefill_report = c.cobrand.prefill_report_fields_for_inspector || c.user.has_body_permission_to('report_prefill') %]
```

Comment:
Current behaviour is it prefilled if `prefill_report_fields_for_inspector` (all UK council cobrands) and user was inspector. This behaviour is now either `prefill_report_fields_for_inspector` or permission, which doesn't match, it'll turn it on for everyone on the council cobrands.
I think it (and same in wrapper) needs to be `(c.cobrand.prefill_report_fields_for_inspector && inspector) || report_prefill` to match.

Ideally, perhaps not this PR, but it should give everyone with inspector permission the prefill permission (in a schema update) and then this line would only be the prefill permission, as you'd expect.

Reply:
Yup. I have put in a test to cover this as automated cobrand testing in the JS isn't really a thing.